webpack/components/Errata/index.js

```diff
@@ -178,3 +188,31 @@ export const ErrataSeverity = ({ severity }) => {
 ErrataSeverity.propTypes = {
   severity: PropTypes.string.isRequired,
 };
+
+export const ErrataToggleGroupItem = ({
+  text, tooltipText, isSelected, onChange,
+}) => (
+  <ToggleGroupItem
+    text={
+      <>
+        {text}
```

Comment:
Will this be translated like this instead of using the `_()`

Reply:
I used the `__` where it's passed into `<ErrataToggleGroupItem>` so I think that should do it! :slightly_smiling_face: